modules/core/04-channel/keeper/keeper.go

```diff
@@ -469,3 +488,8 @@ func (k Keeper) iterateHashes(_ sdk.Context, iterator db.Iterator, cb func(portI
 		}
 	}
 }
+
+// filteredPortPrefix returns the prefix key for the given port prefix.
+func filteredPortPrefix(portPrefix string) []byte {
```

Comment:
Would it make sense to place this function [in `types/keys.go`](https://github.com/cosmos/ibc-go/blob/main/modules/core/04-channel/types/keys.go) instead?